common/component/kafka/metadata.go

```diff
@@ -80,16 +89,23 @@ type KafkaMetadata struct {
 	Version                string              `mapstructure:"version"`
 	internalVersion        sarama.KafkaVersion `mapstructure:"-"`
 	internalOidcExtensions map[string]string   `mapstructure:"-"`
+
+	// configs for kafka client
+	clientConnectionRefreshIntervalMin int `mapstructure:"clientConnectionRefreshIntervalMin"`
+	clientConnectionMaxKeepAliveMin    int `mapstructure:"clientConnectionMaxKeepAliveMin"`
```

Comment:
Please make these time.Duration-we try to avoid durations in seconds or minutes and standardize in supporting Go duration formatting (will also need to update `metadata.yaml`). Also, suggesting better naming

```suggestion
	clientConnectionRefreshInterval time.Duration `mapstructure:"clientConnectionRefreshInterval"`
	clientConnectionKeepAliveInterval    time.Duration `mapstructure:"clientConnectionKeepAliveInterval"`
```